app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from __future__ import annotations
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if masks_to_buy > 0:
+        return f"Friends should buy {masks_to_buy} masks"
```

Comment:
Consider rephrasing the return message to 'Friends should buy masks' for better readability, especially when masks_to_buy is 1. Alternatively, handle singular and plural cases separately.